ChangeLog.md

```diff
@@ -20,6 +20,12 @@ See docs/process.md for more on how version tagging works.
 
 2.0.25
 ------
+- By default (unless `EXIT_RUNTIME=1` is specified) emscripten programs running
+  under node will no longer call `process.exit()` on `exit()`.  Instead they
+  will simply unwind the stack return to the event loop, much like they do
+  on the web.  In many cases the node process will still exit naturally if
+  there is nothing keeping the event loop going.  To guarantee that the node
+  process exits immediately please build with `EXIT_RUNTIME=1`.
```

Comment:
What about error code reporting - is there some chance this causes a test to have a false positive or negative? That is, if it exits immediately, could it swallow an error? I think we have enough checks to avoid that, but I'm not 100% sure.

Reply:
The code path in `quit_` calls `process['exitCode'] = status`.. so even if we exit "naturally" the correct return code will still be generated.

We have a fair bit of testing for this behavior (see `other.test_returncode`) so we know it works and we test a lot of difference circumstances there.